src/routes/jira/server/jira-ghe-server-apps.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraGheServerApps = async (
```

Comment:
Naming convention is to use the method in the name of the function and in the file name. Just noticed that you've previously created `JiraGheServers`. That will need to be updated as well but that one can be done in a separate PR.

Reply:
Will do it in a separate PR.

Reply:
This will be taken care in https://softwareteams.atlassian.net/browse/ARC-1519